package.json

```diff
@@ -57,16 +56,10 @@
     "imagemin-jpegtran": "5.0.2",
     "imagemin-optipng": "5.2.1",
     "imagemin-svgo": "5.2.4",
-    "is-reachable": "2.4.0",
     "lightbox2": "2.10.0",
-    "localtunnel": "1.9.0",
     "lodash.throttle": "4.1.1",
     "postcss-unmq": "1.0.2",
-    "psi": "3.1.0",
-    "require-dir": "1.0.0",
     "sauce-connect-tunnel": "0.2.1",
-    "svg-inline-loader": "0.8.0",
```

Comment:
I removed this as I couldn't find a reference to it in the codebase. This will be needed in the SVG icons per https://github.com/cfpb/cfgov-refresh/pull/4114. Please add it back when it's used and put it in the `devDependencies` or `dependencies`, depending on which environment needs it (development or production). (cc @jimmynotjim)

Reply:
Whoops, good call. I'll make sure to add it back when it's used in the JS modules

Reply:
@anselmbradford the PR that's using this was merged. Can you remove this change please.

Reply:
It's been moved to dev-deps, see https://github.com/cfpb/cfgov-refresh/pull/4120/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R91